src/ILLink.Analyzer/README.md

```diff
@@ -0,0 +1,66 @@
+# ILLink Roslyn analyzer
+
+This is a very early start on a Roslyn analyzer for linker-unsafe patterns. I started with [DiagnosticAnalyzer template](https://github.com/dotnet/roslyn-sdk/tree/master/src/VisualStudio.Roslyn.SDK/Roslyn.SDK/ProjectTemplates/CSharp/Diagnostic/Analyzer) mentioned in [Tutorial: Write your first analyzer and code fix](https://docs.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix).
+
+## Building
+
+```
+> dotnet build illink.sln
+```
+
+For some reason I have trouble packaging this project in the linker source tree. In theory this should work:
```

Comment:
I tracked this down to `IsPackable` property being set to `false`. This is done in https://github.com/mono/linker/blob/7d4feba233c1ed9ab5208edc849c505a03d650ba/Directory.Build.props#L4.
So the simple fix is to add `<IsPackable>true</IsPackable>` to the analyzer project.